tracks/csharp/exercises/isogram/mentoring.md

```diff
@@ -0,0 +1,50 @@
+### Reasonable solutions
+
+#### without using `LINQ` 
+
+```csharp
+using System;
+
+public static class Isogram
+{
+    public static bool IsIsogram(string word)
+    {
+        // Ignore spaces, hyphens, regard uppercase as lower.
+        string trimmedInput = word.Trim().Replace(" ", "").Replace("-", "").ToLower();
+        foreach (char c in trimmedInput)
+        {
+            if (trimmedInput.IndexOf(c) != trimmedInput.LastIndexOf(c))
+                return false;
+        }
+
+        return true;
+    }
+}
+```
+
+#### using `LINQ`
+
+```csharp
+using System;
+using System.Linq;
+
+public static class Isogram
+{
+    public static bool IsIsogram(string word)
+    {
+        // Ignore spaces, hyphens, regard uppercase as lower.
+        var trimmedInput = word.ToLower().ToCharArray().Where(Char.IsLetter);
+        return trimmedInput.Distinct().Count() == trimmedInput.Count();
+    }
+}
+```
+
+### Common suggestions
+
+- In the non LINQ version, try to use a single loop
+
+- Using LINQ next suggestions could be made:
+  * each element of the string should be a letter (in this way we avoid checking for space or '-')
```

Comment:
Good suggestion, maybe a slight rewording:

```suggestion
  * To make things easier, try only processing the string's letters (in this way we avoid checking for spaces or '-').
```

Reply:
This was marked as resolved, but I did not see any change. Was that deliberate?

Reply:
sorry, I have simply missed it. I've just committed it now.

Reply:
No problem! It's easy to miss things when there are a bunch of comments.